src/format_pe.rs

```diff
@@ -491,13 +498,13 @@ fn print_exports(
         }
 
         table.add_row(Row::new(vec![
-            offsetx_cell(entry.offset.unwrap_or(0) as u64),
-            addrx_cell(entry.rva as u64),
-            sz_cell(entry.size as u64),
+            offsetx_cell(u64::try_from(entry.offset.unwrap_or(0))?),
```

Comment:
i'm no sure it's useful to fail in these try_from cases but it's fine

Reply:
I prefer to get an error when integer operations fail instead of a panic. That usually indicates the use of insufficient integer types, or the need for explicit integer operations.